apps/website/docs/infrastructure/monorepository-setup.md

```diff
@@ -0,0 +1,119 @@
+---
+sidebar_label: Creating a Brand-New Mono-Repository on GitHub Using DX Tools
+---
+
+# Creating a Brand-New Mono-Repository on GitHub Using DX Tools
+
+This guide provides step-by-step instructions for creating and setting up a new
+mono-repository on GitHub using DX tools.
+
+:::info
+
+Currently, only projects with a single environment powered by Azure are
+supported. Additional tooling to support other scenarios is under development.
+
+:::
+
+## Repository Setup
+
+Start by creating the repository on GitHub. Then, follow these steps to complete
+the configuration.
+
+### Repository Access Control
+
+Ensure appropriate access control for your repository:
+
+- Grant access to your peers who need it.
```

Comment:
suggestion: link usefule resources to setup up access rights

Reply:
The problem I see here is the link to a private repository

Reply:
you can link the search page ie. https://pagopa.atlassian.net/wiki/search?text=Azure%20-%20Utenti%20gruppi%20e%20ruoli

Reply:
https://github.com/pagopa/dx/pull/391/commits/1e3d7959f6b6c3cb9f22c1f42d29f9a3aaa840e2